### ISSUE TITLE
Is this project still being maintained?

### ISSUE THREAD

**Issue by flynneva** [grbl/grbl#1834]
Last commit was >2 years ago it looks like and it looks like there might have been an effort to spin out the project into a different github organization `gnea`?

What is the status of this repo / the grbl project in general in 2023?

**Comment by breiler** [comment]
No, this repository is no longer maintained. The latest version of GRBL can be found at [gnea](https://github.com/gnea/grbl) but is also not maintained anymore.

GRBL has many forks, but some of the more successful new variants are:
* https://github.com/bdring/FluidNC which uses ESP32 MCU:s
* https://github.com/grblHAL which targets many different MCS:s
